sdk/textanalytics/Azure.AI.TextAnalytics/src/ExtractSummaryAction.cs

```diff
@@ -0,0 +1,76 @@
+    /// an Extractive Text Summarization action in a set of documents.
+    /// For example, set the model version, specify the order in which extracted
+    /// sentences are returned, and more.
+    /// </summary>
+    public class ExtractSummaryAction
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ExtractSummaryAction"/>
+        /// class which allows callers to specify details about how to execute
+        /// an extractive text summarization action in a set of documents.
+        /// For example, set the model version, specify the order in which extracted
+        /// sentences are returned, and more.
+        /// </summary>
+        public ExtractSummaryAction()
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ExtractSummaryAction"/>
+        /// class based on the values of a <see cref="ExtractSummaryOptions"/>.
+        /// It sets the <see cref="ModelVersion"/>, <see cref="DisableServiceLogs"/>,
+        /// <see cref="MaxSentenceCount"/>, and <see cref="OrderBy"/> properties.
+        /// </summary>
+        public ExtractSummaryAction(ExtractSummaryOptions options)
+        {
+            ModelVersion = options.ModelVersion;
+            DisableServiceLogs = options.DisableServiceLogs;
+            MaxSentenceCount = options.MaxSentenceCount;
+            OrderBy = options.OrderBy;
+        }
+
+        /// <summary>
+        /// Gets or sets a value that, if set, indicates the version of the text
+        /// analytics model that will be used to generate the result. For supported
+        /// model versions, see operation-specific documentation, for example:
+        /// <see href="https://docs.microsoft.com/azure/cognitive-services/text-analytics/concepts/model-versioning#available-versions"/>.
+        /// </summary>
+        public string ModelVersion { get; set; }
+
+        /// <summary>
+        /// The default value of this property is 'false'. This means, the Language service logs your input text for 48 hours,
+        /// solely to allow for troubleshooting issues.
+        /// Setting this property to true, disables input logging and may limit our ability to investigate issues that occur.
+        /// <para>
+        /// Please see Cognitive Services Compliance and Privacy notes at <see href="https://aka.ms/cs-compliance"/> for additional details,
+        /// and Microsoft Responsible AI principles at <see href="https://www.microsoft.com/ai/responsible-ai"/>.
+        /// </para>
+        /// </summary>
+        public bool? DisableServiceLogs { get; set; }
+
+        /// <summary>
+        /// The name of this action. If not provided, the service will generate one.
+        /// </summary>
+        public string ActionName { get; set; }
+
+        /// <summary>
+        /// The maximum number of sentences to be returned in the result. The default value is 3.
```

Comment:
I wouldn't document the default. The service could change it. Just say if it's null it will use the service default.